api/pkg/common/common.go

```diff
@@ -35,6 +35,8 @@ const (
 	KLimit  = "limit"
 	KOffset = "offset"
 	KSort   = "sort"
+	KLastModified = "lastmodified"
+	KPrefix = "prefix"
```

Comment:
Filter object by name prfix ,the url request would be like this: /v1/tenantId/bucket?name=like:asw_

Reply:
it's fixed.